fieldservice_account/__manifest__.py

```diff
@@ -15,6 +15,6 @@
         "views/fsm_order.xml",
     ],
     "license": "AGPL-3",
-    "development_status": "Beta",
+    "development_status": "Production/Stable",
```

Comment:
Any reason to move this to Production/Stable?

Reply:
@gbrito-erpopen This module is only needing tests to move to `Production/Stable". Makes sense to me. :+1:

Reply:
As long as it has tests, the module is ready for Production/Stable, correct? clear

Reply:
Correct.